hazelcast-integration/operator-framework/README.md

```diff
@@ -0,0 +1,168 @@
+# Hazelcast Operator
+
+Hazelcast Enterprise is packaged with [Operator Framework](https://github.com/operator-framework), which simplifies deployment on Kubernetes and OpenShift. This is a step-by-step guide how to deploy Hazelcast cluster together with Management Center on your Kubernetes or OpenShift cluster.
+
+## Prerequisites
+
+You must have one of the followings:
+ * Kubernetes cluster (with admin rights) and the `kubectl` command configured (you may use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/))
+ * OpenShift cluster (with admin rights) and the `oc` command configured (you may use [Minishift](https://github.com/minishift/minishift))
+
+## Kubernetes Deployment steps
+
+Below are the steps to start a Hazelcast Enterprise cluster using Operator Framework. Note that the first 3 steps are usually performed only once for the Kubernetes cluster (by the cluster admin). The step 4 is performed each time you want to create a new Hazelcast cluster.
+
+#### Step 1: Create RBAC
+
+Run the following commands to configure the Operator permissions.
+
+    kubectl apply -f rbac.yaml
+
```

Comment:
Here people can hit problems like we saw on GKE? creating admin role binding with gcloud account...

It might be good to create a separate repo in future like Kubernetes engine best practices to list all issues with all k8s services... but that is just an idea...

Reply:
I checked and there are no issues with GKE.